modules/mcuboot/Kconfig

```diff
@@ -39,10 +39,15 @@ config MCUBOOT_IMAGE_VERSION
 	default "0.0.0+0"
 	help
 	  Value to be passed as 'version' argument to 'imgtool.py' when
-	  creating signed image. Note that no semantics are connected to
-	  this variable. It does not provide downgrade prevention, and is only
-	  valuable for debugging purposes. Format: maj.min.rev+build with
-	  latter parts optional.
+	  creating signed image.
+	  Format: maj.min.rev+build with latter parts optional.
```

Comment:
what parts are optional? :)

Reply:
I only formatted the whitespace here :P

But since 2 people complained I will update to say "the build number is optional"